common/dateUtils.js

```diff
@@ -0,0 +1,84 @@
+const R = require('ramda')
+
+const {
+  parse,
+  differenceInMonths,
+  differenceInWeeks,
+  differenceInDays,
+  differenceInHours,
+  format,
+
+  compareDesc,
+  isValid: fnsIsValid,
+} = require('date-fns')
+
+const addTrailingZeroes = (n, length) => {
```

Comment:
what about adding a stringUtils method, paddingLeft?
the method can be simpler like: return ("00000" + n).slice(-5)

Reply:
Ok, can do paddingLeft(str, paddingChar, n)